modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -931,7 +931,22 @@
 			<description>
 				Random range, any floating point value between [code]from[/code] and [code]to[/code].
 				[codeblock]
-				prints(rand_range(0, 1), rand_range(0, 1)) # Prints e.g. 0.135591 0.405263
+				prints(randf_range(0, 1), randf_range(0, 1)) # Prints e.g. 0.135591 0.405263
+				[/codeblock]
+			</description>
+		</method>
+		<method name="randi_range">
+			<return type="int">
+			</return>
+			<argument index="0" name="from" type="int">
+			</argument>
+			<argument index="1" name="to" type="int">
+			</argument>
+			<description>
+				Random range, any 32-bit integer value between [code]from[/code] and [code]to[/code] (inclusive). If [code]to[/code] is lesser than [code]from[/code] they are swapped.
```

Comment:
Are you sure that `to` is inclusive?

You did not copy the documentation for `pcg32_boundedrand_r`, but it seems to be exclusive:
```
// pcg32_boundedrand(bound):
// pcg32_boundedrand_r(rng, bound):
//     Generate a uniformly distributed number, r, where 0 <= r < bound
```

(I'd suggest updating `thirdparty/misc/pcg.cpp` to include the documentation from upstream, it's useful to have.)

Reply:
It's inclusive because the range passed to that function is inclusive - it's not related to the code inside  `pcg32_boundedrand`.

Reply:
Added missing description, however...

Reply:
Ah yes, I see you use `+1` in the `RandomNumberGenerator::randi_range()` implementation.